src/components/v5/common/ActionSidebar/partials/forms/ManageVerifiedMembersForm/hooks.ts

```diff
@@ -0,0 +1,65 @@
+import { Id } from '@colony/colony-js';
+import { useCallback, useMemo } from 'react';
+import { useWatch } from 'react-hook-form';
+import { DeepPartial } from 'utility-types';
+
+import { getManageWhitelistDialogPayload } from '~common/Dialogs/ManageWhitelistDialog/helpers';
+import { useColonyContext } from '~hooks';
+import { ActionTypes } from '~redux';
+import { mapPayload, pipe } from '~utils/actions';
+import { DECISION_METHOD_FIELD_NAME } from '~v5/common/ActionSidebar/consts';
+
+import { DecisionMethod, useActionFormBaseHook } from '../../../hooks';
+import { ActionFormBaseProps } from '../../../types';
+
+import {
+  validationSchema,
+  ManageVerifiedMembersFormValues,
+  AddRemoveMembersType,
+} from './consts';
+
+export const useManageVerifiedMembers = (
+  getFormOptions: ActionFormBaseProps['getFormOptions'],
+) => {
+  const { colony } = useColonyContext();
+  const decisionMethod: DecisionMethod | undefined = useWatch({
+    name: DECISION_METHOD_FIELD_NAME,
+  });
+  const addRemoveMembers: AddRemoveMembersType = useWatch({
+    name: 'addRemoveMember',
+  });
+
+  useActionFormBaseHook({
+    actionType:
+      decisionMethod === DecisionMethod.Permissions
+        ? ActionTypes.ACTION_VERSION_UPGRADE
+        : ActionTypes.ROOT_MOTION,
+    getFormOptions,
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    transform: useCallback(
+      pipe(
+        // @TODO: Fix types when form will be wired up with API
+        mapPayload((values: any) => {
+          if (!colony) {
+            return null;
+          }
+
+          return getManageWhitelistDialogPayload(
```

Comment:
we can stub this out or jsut write a comment because we will not be using this function